index.js

```diff
@@ -43,7 +45,13 @@ module.exports = {
           passed: passed,
           errorMessage: relativePath + ' should pass ESLint.' + errors
         }]);
-      }
+      },
+
+      // Find broccoli-lint-eslint's babel-eslint parser and use it as our default
+      options: {
+        parser: path.dirname(resolve.sync('broccoli-lint-eslint/node_modules/babel-eslint')),
```

Comment:
Should not embed `node_modules` here like this.  If you need to resolve from another relative location, you have to build up the path.